### DISCUSSION TITLE
Analogues to Ranger's `--choosefiles` and `--choosedir`?

### DISCUSSION THREAD

**Discussion by poperigby** [sxyazi/yazi#1128] [1. Q&A] [answered]
Ranger has these two flags:

```
--choosefiles=OUTFILE Makes ranger act like a file chooser for multiple
                      files at once. When opening a file, it will quit and
                      write the name of all selected files to OUTFILE.

--choosedir=OUTFILE   Makes ranger act like a directory chooser. When ranger
                      quits, it will write the name of the last visited
                      directory to OUTFILE
```

Does Yazi have something similar?

I'm trying to setup a wrapper script for [this](https://github.com/boydaihungst/xdg-desktop-portal-termfilechooser)

**Comment by sxyazi** [comment, marked as answer]
Yazi has a `--chooser-file` option similar to `--choosefiles`, and `--choosedir` is not supported atm.

You can check more available options by running `yazi --help`.

**Reply by poperigby** [reply]
Thank you @sxyazi. Should I add a comment to #51 about a `--choosedir` alternative?
